api/proto/metrics/metrics.proto

```diff
@@ -20,8 +20,40 @@ enum ZmetricTypes {
 message memoryMetric {
   uint32 usedMem = 2;            //in MBytes
   uint32 availMem = 3;           //in MBytes
-  double usedPercentage = 4;
-  double availPercentage = 5;
+  double usedPercentage = 4; // Obsolete - No Longer Used.
+  double availPercentage = 5; // Obsolete - No Longer Used.
+}
+
+// App Memory Metric
+message AppMemoryMetric {
+    // totalMB - Memory allocated to app instance by EVE. In Steady State,
+    //  this is expected to be same as in App instance config. Eve should
+    //  report what it actually allocated / reserved instead of what is configured.
+    uint32 totalMB = 1;
+
+    // usedMB - Memory Used by app from totalMB Memory
+    //  usedMB <= totalMB in normal cases.
+    //  But usedMB can exceed totalMB - this could be a warning to the user.
```

Comment:
I'd say 'the enforcement of the allocated could be lax hence used can sometimes exceed allocated.

Reply:
Done:

    // usedMB - Memory Used by app from totalMB Memory
    //  usedMB <= totalMB in normal cases.
    //  Enforcement of the allocatedMB could be lax, hence usedMB can sometimes
    //   exceed allocatedMB.
    uint32 usedMB = 2;